spec/requests/services_spec.rb

```diff
@@ -318,72 +318,116 @@ def format_retirement_date(time)
+      it "supports single service retirement now" do
+        api_basic_authorize collection_action_identifier(:services, :retire)
 
-      expect(MiqEvent).to receive(:raise_evm_event).once
+        expect(MiqEvent).to receive(:raise_evm_event).once
 
-      post(api_service_url(nil, svc), :params => gen_request(:retire))
+        post(api_service_url(nil, svc), :params => gen_request(:retire))
 
-      expect_single_resource_query("id" => svc.id.to_s, "href" => api_service_url(nil, svc))
-    end
+        expect_single_resource_query("id" => svc.id.to_s, "href" => api_service_url(nil, svc))
+      end
 
-    it "supports single service retirement in future" do
-      api_basic_authorize collection_action_identifier(:services, :retire)
+      it "supports single service retirement in future" do
+        api_basic_authorize collection_action_identifier(:services, :retire)
 
-      ret_date = format_retirement_date(Time.now + 5.days)
+        ret_date = format_retirement_date(Time.zone.now + 5.days)
 
-      post(api_service_url(nil, svc), :params => gen_request(:retire, "date" => ret_date, "warn" => 2))
+        post(api_service_url(nil, svc), :params => gen_request(:retire, "date" => ret_date, "warn" => 2))
 
-      expect_single_resource_query("id" => svc.id.to_s, "retires_on" => ret_date, "retirement_warn" => 2)
-      expect(format_retirement_date(svc.reload.retires_on)).to eq(ret_date)
-      expect(svc.retirement_warn).to eq(2)
-    end
+        expect_single_resource_query("id" => svc.id.to_s, "retires_on" => ret_date, "retirement_warn" => 2)
+        expect(format_retirement_date(svc.reload.retires_on)).to eq(ret_date)
+        expect(svc.retirement_warn).to eq(2)
+      end
 
-    it "supports multiple service retirement now" do
-      api_basic_authorize collection_action_identifier(:services, :retire)
+      it "supports multiple service retirement now" do
+        api_basic_authorize collection_action_identifier(:services, :retire)
 
-      expect(MiqEvent).to receive(:raise_evm_event).twice
+        expect(MiqEvent).to receive(:raise_evm_event).twice
 
-      post(api_services_url, :params => gen_request(:retire,
-                                                    [{"href" => api_service_url(nil, svc1)},
-                                                     {"href" => api_service_url(nil, svc2)}]))
+        post(api_services_url, :params => gen_request(:retire,
+                                                      [{"href" => api_service_url(nil, svc1)},
+                                                       {"href" => api_service_url(nil, svc2)}]))
+
+        expect_results_to_match_hash("results", [{"id" => svc1.id.to_s}, {"id" => svc2.id.to_s}])
+      end
+
+      it "supports multiple service retirement in future" do
+        api_basic_authorize collection_action_identifier(:services, :retire)
+
+        ret_date = format_retirement_date(Time.zone.now + 2.days)
```

Comment:
I take it these updates were for rubocop ?

Reply:
Yup!